src/helpers/TouchEventManager.js

```diff
@@ -67,8 +74,11 @@ const TouchEventManager = {
           zoom: core.getZoom(),
           type: isDoubleTap ? 'doubleTap' : 'tap',
           touchStartTimeStamp: Date.now(),
-          stopMomentumScroll: true
+          stopMomentumScroll: true,
+          touchMoveCount: 0,
         };
+        this.startingScrollLeft = this.container.scrollLeft;
```

Comment:
This variable sounds like a boolean to me... Is there any reason we are not using names like `this.containerScrollLeft`?

Reply:
oh I see the reason, nvm

Reply:
I'll update the name, but I do want "starting" in it name because it gets set during 'handleTouchStart' and doesn't get changed (until the next handleTouchStart).  

Maybe ' containerStartingScrollLeft' but that a bit long, how about I move it to the 'this.touch' object and rename it to 'startingPositionX'